src/webportal/src/app/user/user-logout/user-logout.component.js

```diff
@@ -36,4 +36,23 @@ const userLogout = (origin = window.location.href) => {
   }
 };
 
-module.exports = { userLogout };
+/**
+ * Clear local tokens only, will not redirect user to oidc logout page
+ * @description Will try to refresh the token if oidc enabled.
+ * @param {string} origin - redirect target after login
+ */
+const clearToken = (origin = window.location.href) => {
+  cookies.remove('user');
+  cookies.remove('token');
+  cookies.remove('admin');
+  cookies.remove('my-jobs');
+  if (!origin) {
+    window.location.replace('/index.html');
+  } else {
+    window.location.replace(
+      `/index.html?${querystring.stringify({ from: origin })}`,
+    );
+  }
+};
+
+module.exports = { userLogout, clearToken };
```

Comment:
revoke current browser token in `userLogout`? otherwise, every login/logout will create one new token

Reply:
All exists tokens will be verified when rest server handles token operations (login, create, revoke) and the expired tokens will be cleaned. So don't worry about that there will be too many tokens stored in k8s secret.

Besides, the idea is great. As the release date is approaching, I prefer to merge this pull request first and I will implement the logic that revoke tokens when user logout / change password in another pull request.